test/lint/test_runner/src/main.rs

```diff
@@ -292,6 +292,33 @@ fn lint_doc() -> LintResult {
     }
 }
 
+fn lint_markdown() -> LintResult {
+    let bin_name = "mlc";
+    let mut md_ignore_paths = get_subtrees();
+    md_ignore_paths.push("./doc/README_doxygen.md");
+    let md_ignore_path_str = md_ignore_paths.join(",");
+
+    let mut cmd = Command::new(bin_name);
+    cmd.args([
+        "--offline",
+        "--ignore-path",
+        md_ignore_path_str.as_str(),
+        "--root-dir",
+        ".",
+    ])
+    .stdout(Stdio::null()); // Suppress overly-verbose output
+
+    match cmd.status() {
+        Ok(status) if status.success() => Ok(()),
+        Ok(_) => Err("mlc command failed".to_string()), // Ran but did not succeed
```

Comment:
Nit: It would be nice to have a message that briefly describes and motivates the lint check as elsewhere

e.g. from `lint_std_filesystem` 
```rust
    if found {
        Err(r#"
^^^
Direct use of std::filesystem may be dangerous and buggy. Please include <util/fs.h> and use the
fs:: namespace, which has unsafe filesystem functions marked as deleted.
            "#
        .to_string())
```

Reply:
OK I updated the error message.

Previously:

```
[Err ] ./.venv/lib/python3.9/site-packages/mypyc/external/googletest/README.md (57, 1) => CMakeLists.txt - Target filename not found.
[Err ] ./.venv/lib/python3.9/site-packages/mypyc/external/googletest/README.md (280, 10) => docs/PumpManual.md - Target filename not found.
[Err ] ./.venv/lib/python3.9/site-packages/mypyc/external/googletest/README.md (279, 47) => scripts/ - Target not found.

The following links could not be resolved:
mlc command failed
^---- ⚠️  Failure generated from markdown hyperlink check!
```

On 838eacec6a5e31c4a7fd6f9c95510e275f27121a:

```
One or more markdown links are broken.

Currently, relative links are preferred (but not required) as jumping to file works natively within
Emacs.

Markdown link errors found:
[Err ] ./.venv/lib/python3.9/site-packages/mypyc/external/googletest/README.md (57, 1) => CMakeLists.txt - Target filename not found.
[Err ] ./.venv/lib/python3.9/site-packages/mypyc/external/googletest/README.md (279, 47) => scripts/ - Target not found.
[Err ] ./.venv/lib/python3.9/site-packages/mypyc/external/googletest/README.md (280, 10) => docs/PumpManual.md - Target filename not found.


^---- ⚠️  Failure generated from markdown hyperlink check!
```